### PR TITLE
Build Tool Setup

### PR DESCRIPTION
**Pull request by pjayme**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "kiwibuild",
+  "version": "1.0.0",
+  "engines": {
+    "node": "^8.0",
+    "npm": "^5.0"
```
Comment (Irsanarisandy) [inline package.json]: For node and npm, do we not need the latest ones, i.e. node 10.15 (LTS) and npm 6.4 ?

**Comment by ishannz** [comment]
@pjayme Add the base Page.ss file in the templates folder as well.

File: package.json (hunk 1 above)
Comment (pjayme) [reply package.json]: sweet, have updated all modules to latest versions

File: .editorconfig [hunk 2]
```diff
@@ -15,9 +15,12 @@ trim_trailing_whitespace = true
 [*.md]
 trim_trailing_whitespace = false
 
-[*.{yml,js,json,css,scss,eslintrc}]
+[*.{yml,js,scss,ss,vue}]
 indent_size = 2
```
Comment (Irsanarisandy) [inline .editorconfig]: If you don't mind, can you set the indent size to 4? I find 2 to be too small for indentation.

Comment (pjayme) [reply .editorconfig]: Most SS projects use 2 by convention :)

Comment (Irsanarisandy) [reply .editorconfig]: Ah, ok

**Pull request merged**